app/javascript/controllers/patron_request_controller.js

```diff
@@ -18,6 +18,21 @@ export default class extends Controller {
     }
   }
 
+  sort(event) {
+    const sortby = event.target.dataset.patronrequestSortfield;
+    const isAsc = event.target.dataset.patronrequestAsc == 'true' ? true : false;
+    event.target.dataset.patronrequestAsc = !isAsc;
```

Comment:
I think stimulus would like use to use their values to manage the current state:
https://stimulus.hotwired.dev/handbook/managing-state

I wonder if the item selector controller (or a new one?) would be a better place to manage this behavior